src/Resources/config/routing-dynamic.xml

```diff
@@ -64,6 +64,10 @@
             <call method="setRequestStack"><argument type="service" id="request_stack"/></call>
         </service>
 
+        <service id="cmf_routing.redirectable_dynamic_router" class="Symfony\Cmf\Bundle\RoutingBundle\Routing\RedirectableUrlMatcher" decorates="cmf_routing.dynamic_router">
```

Comment:
as i am not super familiar with the service decoration functionality: is this enough to make a symfony application that installs the bundle use the redirecting matcher?

and is the decorates feature supported in all symfony versions that this bundle supports?

Reply:
looking at https://github.com/symfony-cmf/routing-bundle/pull/465/checks?check_run_id=2846066414 this seems not to work exactly like this.